Signal/src/ViewControllers/CodeVerificationViewController.m

```diff
@@ -65,7 +66,13 @@ - (instancetype)init
 
 - (void)viewDidLoad {
     [super viewDidLoad];
-    
+
+    self.navigationItem.backBarButtonItem =
+        [[UIBarButtonItem alloc] initWithTitle:NSLocalizedString(@"BACK_BUTTON", @"button text for back button")
```

Comment:
Why can't we use a normal back button?

Reply:
You're right - we need this in the app settings (where the default back button labels are way too long), but not here.  Removed.